templates/3a_monitor_MIBI_run.ipynb

```diff
@@ -1,20 +1,13 @@
 {
```

Comment:
<p>Looks good, just remove the details about intensities and tim_res, which the user won't ever need to change</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>